### ISSUE TITLE
🐛 Bug Report: Accessibility issue on profile dropdown

### ISSUE THREAD

**Issue by NiazMorshed2007** [appwrite/appwrite#5724]
### 👟 Reproduction steps

When I click the profile dropdwon trigger on header,
It opens the dropdown but I accessibility of it using keyboard doesn't work.
Here is a video showing the problem. I clicked down and up arrow key,, but doesn't select any dropdown item and scrolls through the page.

https://github.com/appwrite/appwrite/assets/77217706/8edeb90e-e337-4bd8-8c73-9679fff0ecf4

### 👍 Expected behavior

Should have accessibility with keyboard.

### 👎 Actual Behavior

Not have accessibility 

### 🎲 Appwrite version

Appwrite Cloud

### 💻 Operating system

Windows

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by joeyouss** [comment]
Hey there, with keyboard, can you try using the `tab` key please once?

**Comment by joeyouss** [comment]
Please consider enabling caret browsing. It appears that this issue does not originate from Appwrite, so I suggest closing this issue. I strongly encourage you to review your system settings and/or utilize the tab key for navigation.
